toolset/setup/docker/databases/mysql/my.cnf

```diff
@@ -42,6 +42,12 @@ table_definition_cache  = 800
 max_heap_table_size     = 128M
 tmp_table_size          = 128M
 
+#
+# monitoring off
+#
+
+performance-schema = false
```

Comment:
Can we get some documentation explaining what this does?

Reply:
https://mariadb.org/maria-10-1-mysql-5-7-commodity-hardware/
https://github.com/TechEmpower/FrameworkBenchmarks/issues/2845#issuecomment-363475071

Reply:
More information: 
https://dev.mysql.com/doc/refman/5.7/en/performance-schema.html
In production is not useful and the performance will be better.